pkg/thing/mocks/logger.go

```diff
@@ -0,0 +1,15 @@
+package mocks
+
+type FakeLogger struct{}
+
+func (fl *FakeLogger) Info(...interface{}) {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method FakeLogger.Info should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)